ch_pipeline/hfb/io.py

```diff
@@ -252,9 +241,7 @@ def _find_beam(self):
         beams_xy = mdl.get_beam_positions(beams_ind, freq).squeeze()
 
         # Find NS beam number of beam closest to calibration source
-        beam_index_ns = np.abs(beams_xy[:, 1] - src_y).argmin()
-
-        return beam_index_ns
+        return np.abs(beams_xy[:, 1] - src_y).argmin()
```

Comment:
It seems to me something useful gets lost when not allowing the result of a calculation to be stored in a variable before returning it:
- The name of the variable conveys some information.
- The `Returns` section in the docstring refers to this variable.

I guess this is an opinion of the ruff linter that I'll just have to get used to (unnecessary-assign (RET504))